src/OpenTelemetry.Api/README.md

```diff
@@ -70,7 +71,39 @@ with the intent to produce continuous summaries of those measurements.
 
 _**Warning:** OpenTelemetry .NET has a prototype Metrics API implementation only
 and is not recommended for any production use. The API is expected to change
-heavily. Please check the [Metric support plan](https://github.com/open-telemetry/opentelemetry-dotnet/issues/1501)._
+heavily. Please check the [Metric support
+plan](https://github.com/open-telemetry/opentelemetry-dotnet/issues/1501)._
+
+### Baggage API
+
+[Baggage
+API](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/baggage/api.md)
+allows users to add context to metric, traces, and logs. Baggage information is
+automatically propagated by [Propagators
```

Comment:
"automatically propagated" could be misleading, as user can set Propagators to be just TraceContext which won't propagate Baggage..  We could say Baggage **can be** propagated out of proc using Propagators. And say Otel SDK ships a BaggagePropagator and enables it by default.

Reply:
Fixed this.